index.html

```diff
@@ -60,26 +60,31 @@
 
-	const $ = document.querySelector.bind(document);
+	  const $ = document.querySelector.bind(document);
+    function init() {
+        const names = Object.keys(songs);
+        const random = arr => arr[Math.floor(Math.random() * arr.length)];
+        const $answer = $('#answer');
+        const $lyric = $('#lyric');
+        let answer = '';
```

Comment:
Lexical scope is dumb. var works beautifully if you divorce yourself from C style languages and embrace function level scope.